### ISSUE TITLE
Multiple genome files reported to contain no sequence

### ISSUE THREAD

**Issue by drhoads** [johnlees/PopPUNK#199]
**Versions**

**Command used and output returned**

**Describe the bug**

Poppunk v 2.4.0
sketchlib v 1.7.3
Running in a WSl2-Ubuntu VM on Win10.
I have seen this error before and it often had to do with malformed strain names with space or ( in them or when I forgot to convert my r-list.txt with dos2unix.  The same installation and bash file is used 
bash file:*************
poppunk --create-db --output PoppunkDB --r-files r-list.txt --external-clustering external_clusters.csv --qc-filter continue --threads 8
poppunk --fit-model lineage --ref-db PoppunkDB --output PopPunkLineageModel --graph-weights --qc-filter continue  --threads 8
poppunk_visualise --ref-db PoppunkDB --model-dir PopPunkLineageModel --output Poppunk_viz --overwrite --microreact  --threads 8
***************
All the requisite genomes are first copied into a subfolder 'genomes' and then build the r-list.txt
The bash finishes just fine on one set of 44 genomes, but then for a set of 37 (many of them the same) I get errors that multiple files contain no sequence, then a series of traceback calls:
Graph-tools OpenMP parallelisation enabled: with 8 threads
Mode: Building new database from input sequences
Sketching 37 genomes using 8 thread(s)
Progress (CPU): 37 / 37
SP88d contains no sequence
B3-14B contains no sequence
SP82a contains no sequence
1510 contains no sequence
ch5 contains no sequence
ED98 contains no sequence
1521 contains no sequence
1516 contains no sequence

Traceback (most recent call last):
  File "/home/dougrhoads/miniconda3/envs/Poppunk/bin/poppunk", line 11, in <module>
    sys.exit(main())
  File "/home/dougrhoads/miniconda3/envs/Poppunk/lib/python3.9/site-packages/PopPUNK/__main__.py", line 307, in main
    constructDatabase(
  File "/home/dougrhoads/miniconda3/envs/Poppunk/lib/python3.9/site-packages/PopPUNK/sketchlib.py", line 407, in constructDatabase
    pp_sketchlib.constructDatabase(dbname,
RuntimeError: Errors during sketching
PopPUNK (POPulation Partitioning Using Nucleotide Kmers)
        (with backend: sketchlib v1.7.3
         sketchlib: /home/dougrhoads/miniconda3/envs/Poppunk/lib/python3.9/site-packages/pp_sketchlib.cpython-39-x86_64-linux-gnu.so)
************
Note that I have used these same genomes before with success.  I even tried running dos2unix on all the .fna files
[r-list.txt](https://github.com/johnlees/PopPUNK/files/8149086/r-list.txt)
.

**Comment by drhoads** [comment]
Please disregard this issue posting and close this.  I missed a couple details.  Finally figured it out.  I had lost my old notes.  I will try to not make that mistake again.

**Comment by nickjcroucher** [comment]
Thanks for letting us know @drhoads - happens to all of us - please reopen if you want to let us know what the cause was, for the reference of anyone else encountering this error in the future

**Comment by drhoads** [comment]
Well, there were several issues, and most of them were bone-head moves on my part.  the files were properly transferred and the r-list.txt was prepared.  Then I forgot that I needed to include the actual PATH to the copied files and switching back and forth from DOS and Linux I used the wrong folder symbol "\" instead of "/".  So in REALITY I was looking for the wrong problem (what is wrong with the sequence file) because the program said there was no sequence data in the file, but in reality the program could not FIND the file because I had given an incorrect path.
